pkg/cli/types.go

```diff
@@ -51,7 +51,7 @@ type validator = func(val interface{}) error
 
 // CommandLineInterface is a type to group CLI funcs and state
 type CommandLineInterface struct {
-	rootCmd       *cobra.Command
+	Command       *cobra.Command
```

Comment:
Why change to public?

Reply:
so that this command can be used in a larger cobra app